### PR TITLE
Next previous

### PR DESCRIPTION
**Pull request by imomaliev**

### CODE REVIEW

File: ordered_model/models.py [hunk 1]
```diff
@@ -58,25 +58,29 @@ def get_ordering_queryset(self, qs=None):
...
     def previous(self):
         """
         Get previous element in this object's ordered stack.
         """
-        try:
-            return self.get_ordering_queryset().filter(**{self.order_field_name + '__lt': getattr(self, self.order_field_name)}).order_by('-' + self.order_field_name)[0]
-        except IndexError:
-            # first
-            return None
+        return self._previous_qs.first()
```
Comment (Speedy1991) [inline ordered_model/models.py]: Tests are failing because curved brackets hare missing here:
`return self._previous_qs().first()`

**Comment by imomaliev** [comment]
@b

**Comment by imomaliev** [comment]
@bfirsh Ready for review

**Comment by bfirsh** [comment]
Awesome! LGTM! Thanks all.

**Comment by Speedy1991** [comment]
@bfirsh is there any date for the new release with this PR? :)

**Comment by imomaliev** [comment]
@Speedy1991 Hi, I was planning releasing this when https://github.com/bfirsh/django-ordered-model/pull/171 will be ready? But if you really need this functionality I can make a release

**Comment by Speedy1991** [comment]
Thanks for the feedback.
It's ok for me to wait until #171 is done. Thanks for this awsome feature + this package at all

**Comment by Speedy1991** [comment]
Friendly Ping @imomaliev 
Any news for a release date since #171 is merged?

**Comment by imomaliev** [comment]
@Speedy1991 Checkout new 3.1.0 release

**Comment by Speedy1991** [comment]
Awesome! Thank you really much :)

**Pull request merged**